cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: che-server
```

Comment:
I don't see why a file needs to be added in a repository for building it on a CI server.
If each CI needs to add config files in the repository, it will be a mess.